web/modules/custom/joinup_user/joinup_user.module

```diff
@@ -342,8 +488,24 @@ function joinup_user_og_membership_presave(EntityInterface $entity) {
 }
 
 /**
- * Implements hook_ENTITY_TYPE_predelete() for OG Membership entities.
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
  */
-function joinup_user_og_membership_predelete(EntityInterface $entity) {
-  joinup_user_og_membership_presave($entity);
+function joinup_user_form_user_multiple_cancel_confirm_alter(&$form, FormStateInterface $form_state, $form_id) {
+  joinup_user_form_user_cancel_form_alter($form, $form_state, $form_id);
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
+ */
+function joinup_user_form_user_cancel_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
Also, as for the read-only config, this form is not storing anything in the database and the config storage settings for the user deletion serve only as a default value when the user has the 'administer users' permission.